src/firstvoices/backend/models/characters.py

```diff
@@ -143,21 +103,123 @@ def __str__(self):
         return self.title
 
 
-# See fw-4172: AlphabetMapper model implementation once g2p work has been completed
-# class AlphabetMapper(BaseSiteContentModel):
-#     """
-#     Represents a private table holding the g2p mappings and configuration for a site.
-#     """
-#
-#     class Meta:
-#         verbose_name = _("alphabet mapper")
-#         verbose_name_plural = _("alphabet mappers")
-#
-#     # See fw-4172: Once file storage is configured, enable these FileFields / change to proper paths
-#     # alphabet_mapper is a placeholder for now.
-#     # input_to_canonical = models.FileField(upload_to="alphabet_mapper")
-#     # canonical_to_base = models.FileField(upload_to="alphabet_mapper")
-#     # g2p_config = models.FileField(upload_to="alphabet_mapper")
-#
-#     def __str__(self):
-#         return self.id
+class AlphabetMapper(BaseSiteContentModel):
```

Comment:
This has enough different properties and methods that I wonder if it would make sense to call it just Alphabet